### PR TITLE
Fix: Set aria-expanded attribute true/false to match drawer visibility

### PR DESCRIPTION
**Pull request by cahirodoherty-learningpool**
[//]: # (Please title your PR according to eslint commit conventions)
[//]: # (See https://github.com/conventional-changelog/conventional-changelog/tree/master/packages/conventional-changelog-eslint#eslint-convention for details)

[//]: # (Link the PR to the original issue)
Fixes: #409 

[//]: # (Delete Fix, Update, New and/or Breaking sections as appropriate)
### Fix
* Set the aria-expanded attribute appropriately on the drawer

### CODE REVIEW

File: js/views/drawerView.js [hunk 1]
```diff
@@ -110,7 +111,10 @@ class DrawerView extends Backbone.View {
 
   showDrawer(emptyDrawer, position = null) {
     this.setDrawerPosition(position);
-    this.$el.removeClass('u-display-none').removeAttr('aria-hidden');
+    this.$el
+      .removeClass('u-display-none')
+      .removeAttr('aria-hidden')
```
Comment (lemmyadams) [inline js/views/drawerView.js:116]: Should this set aria-hidden to false now instead of removing it? To match what's happening with aria-expanded.

**Review by lemmyadams — APPROVED** [review]

**Review by ethan-lp — APPROVED** [review]

File: js/views/drawerView.js [hunk 2]
```diff
@@ -17,7 +17,8 @@ class DrawerView extends Backbone.View {
       role: 'dialog',
       'aria-modal': 'true',
       'aria-labelledby': 'drawer-heading',
-      'aria-hidden': 'true'
+      'aria-hidden': 'true',
+      'aria-expanded': 'false';
```
Comment (kirsty-hames) [inline js/views/drawerView.js:21]: Is the semicolon needed here?

Otherwise working as expected thanks. Testing on Mac Safari running Voiceover.

**Comment by kirsty-hames** [comment]
Related to this PR...
PLP - https://github.com/adaptlearning/adapt-contrib-pageLevelProgress/issues/188
Language Picker - https://github.com/adaptlearning/adapt-contrib-languagePicker/issues/84

**Review by kirsty-hames — APPROVED** [review]

**Pull request merged**